modules/apps/27-interchain-accounts/controller/types/errors.go

```diff
@@ -6,5 +6,7 @@ import (
 
 // ICA Controller sentinel errors
 var (
-	ErrControllerSubModuleDisabled = sdkerrors.Register(SubModuleName, 2, "controller submodule is disabled")
+	ErrControllerSubModuleDisabled = sdkerrors.Register(SubModuleName, 1, "controller submodule is disabled")
+	ErrInvalidTimeout              = sdkerrors.Register(SubModuleName, 2, "timeout height and timestamp cannot both be zero")
```

Comment:
```suggestion
	ErrInvalidTimeout              = sdkerrors.Register(SubModuleName, 2, "invalid timeout")
```

We should probably add an error type in 04-channel for this if it doesn't already exist (followup pr). I suggest you change the error message since in your usage of it, the error message would be stated twice:
> msg timeout height and msg timeout timestamp cannot both be 0: timeout height and timestamp cannot both be zero